source/index.ts

```diff
@@ -492,95 +496,95 @@ export const enum AssertionTypeDescription {
 // Type assertions have to be declared with an explicit type.
 type Assert = {
 	// Unknowns.
-	undefined: (value: unknown) => asserts value is undefined;
-	string: (value: unknown) => asserts value is string;
-	number: (value: unknown) => asserts value is number;
-	bigint: (value: unknown) => asserts value is bigint;
+	undefined: (value: unknown, customMessage?: string) => asserts value is undefined;
```

Comment:
```suggestion
	undefined: (value: unknown, message?: string) => asserts value is undefined;
```

I don't think the "custom" part is needed.